server/src/main/java/org/elasticsearch/action/ResolvedIndexExpression.java

```diff
@@ -75,9 +76,11 @@ public enum LocalIndexResolutionResult {
 
     /**
      * Represents local (non-remote) resolution results, including expanded indices, and a {@link LocalIndexResolutionResult}.
+     *
+     * @param indices represents the resolved concrete indices backing the expression
```

Comment:
I am ok with the field to be renamed to `indices` since it is better than `expressions` and is actually more consistent with the other field name `localIndexResolutionResult`. But besides concrete indices, they can also be data stream names, aliases. Can we make it clear in this comment?